r/tests/testthat/test-dplyr-funcs-string.R

```diff
@@ -1103,6 +1130,14 @@ test_that("str_starts, str_ends, startsWith, endsWith", {
     df
   )
 
+  compare_dplyr_binding(
+    .input %>%
+      transmute(a = str_ends(x, "r"),
+                b = str_ends(x, "r", negate = TRUE),
+                d = str_ends(x, fixed("r"))) %>%
```

Comment:
```suggestion
                b = str_ends(x, "r", negate = TRUE),
                c = str_ends(x, fixed("r")),
                d = str_ends(x,  fixed("r"), negate = TRUE)) %>%
```

I think this would catch what I was thinking up above?

Reply:
Perfect thanks!